sdk/template/ci.yml

```diff
@@ -52,3 +52,12 @@ extends:
     Artifacts:
     - name: Azure.Template
       safeName: AzureTemplate
+      triggeringPaths:
```

Comment:
Do we need these now with the fallback? I would expect we don't.

Reply:
What? We removed `ci.yml` triggers in favor of just these.

Reply:
I've come back to this after reading it again. You mean

> with the addition of the template packages if nothing else matches, why should we add this

Fair point. I guess I just prefer to be explicit. I should probably just yank this then.

Reply:
Yes I'm asking specifically about the path triggers for template. I don't really like the idea of trying to maintain a list of all directories and/or files that are not sdk here. That is part of why I wanted the fallback to always add template if we don't find any other package.